akka-persistence-typed/src/main/scala/akka/persistence/typed/SideEffect.scala

```diff
@@ -58,5 +62,10 @@ object SideEffect {
     new Callback(s ⇒ callback.apply(s))
 
   def stop[State](): SideEffect[State] = Stop.asInstanceOf[SideEffect[State]]
+
+  /**
+   * Unstash the commands that were stashed with `javadsl.EffectFactories.stash` or `scaladsl.Effect.stash`
+   */
+  def unstashAll[State](): SideEffect[State] = UnstashAll.asInstanceOf[SideEffect[State]]
```

Comment:
I just think these functions having `()` are misleading as `def func[A]():SideEffect[A]` because no parameters are accepted. I realize it's done everywhere.

Reply:
Using `()` to indicate that when running the effects they have side effects, in this case the messages in the stash buffer are processed